shared/tests/test_prompts.py

```
@@ -84,3 +84,143 @@ def test_kb_prompt_no_rag_contains_required_content(self):
+
+    def test_empty_string(self):
+        from shared.prompts.constants import parse_prompt_blocks
+
+        text_content, extra = parse_prompt_blocks("")
+        assert text_content == ""
+        assert extra == []
+
+    def test_importable_from_init(self):
+        from shared.prompts import parse_prompt_blocks
+
+        text, extra = parse_prompt_blocks("test")
```

Ruff (0.15.6)

[warning] Unpacked variable `extra` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)